### PR TITLE
Add some manual typing changes

### PR DESCRIPTION
**Pull request by DanielNoord**
- [x] Add yourself to CONTRIBUTORS if you are a new contributor.
- [x] Write a good description on what the PR does.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :hammer: Refactoring   |

## Description

Ref https://github.com/PyCQA/pylint/issues/4683.

### CODE REVIEW

File: pylint/checkers/variables.py [hunk 1]
```diff
@@ -516,7 +516,7 @@ class ScopeConsumer(NamedTuple):
 
     to_consume: dict[str, list[nodes.NodeNG]]
     consumed: dict[str, list[nodes.NodeNG]]
-    consumed_uncertain: DefaultDict[str, list[nodes.NodeNG]]
+    consumed_uncertain: collections.defaultdict[str, list[nodes.NodeNG]]
```
Comment (cdce8p) [inline pylint/checkers/variables.py:519]: We can use `defaultdict` directly. Even if we already import `collections`.
```py
from collections import defaultdict
```

Comment (cdce8p) [inline pylint/checkers/variables.py:519]:
```suggestion
    consumed_uncertain: defaultdict[str, list[nodes.NodeNG]]
```

**Review by Pierre-Sassoulas — APPROVED** [review]
I agree with @cdce8p that collections.DefaultDict is a little too verbose, otherwise LGTM.

**Review by cdce8p — APPROVED** [review]

**Comment by DanielNoord** [comment]
Thanks @cdce8p for fixing and merging this. I was away this evening!

**Comment by cdce8p** [comment]
> Thanks @cdce8p for fixing and merging this. I was away this evening!

No problem! I had checked out the branch locally anyway, so it wasn't a big deal.

**Comment by DanielNoord** [comment]
And thanks for the many reviews today! This could have taken a couple of days of back-and-forth if you hadn't been so active today! Much appreciated 😄

**Comment by cdce8p** [comment]
> And thanks for the many reviews today! This could have taken a couple of days of back-and-forth if you hadn't been so active today! Much appreciated 😄

It's you who did the majority of the work while also fixing a regression! Thank you 🚀

_Now we just need to do that again for astroid_ 😅

**Pull request merged**